src/main/scala/com/fulcrumgenomics/sv/tools/SvPileup.scala

```diff
@@ -89,9 +89,10 @@ class SvPileup
                                  tracker: BreakpointTracker,
                                  writer: SamWriter): Unit = {
     if (evidences.nonEmpty) {
-      val bps = evidences.map(e => tracker(e.breakpoint).id).toSet.toSeq.sorted.mkString(",")
-      val evs = evidences.map(_.evidence.snakeName).mkString(",")
       template.allReads.foreach { rec =>
+        val readEvs = evidences.filter(_.recs.contains(rec))
+        val bps     = readEvs.map(e => tracker(e.breakpoint).id).toSet.toSeq.sorted.mkString(",")
+        val evs     = readEvs.map(_.evidence.snakeName).mkString(",")
```

Comment:
rather than adding the tags to all reads, now only the reads that observed a given break point